hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/CompressionContext.java

```diff
@@ -18,37 +18,117 @@
 
 package org.apache.hadoop.hbase.regionserver.wal;
 
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.util.EnumMap;
 import java.util.Map;
 
```

Comment:
delete this newline and you will have completely cleaned up the import checkstyle warnings. Also, if you were unaware, you can have IDEA do this for you automatically when you "Optimize Imports" by using the checkstyle plugin and importing our checkstyle configuration settings. I documented this a while back ; see [our book](http://hbase.apache.org/book.html#_intellij_idea).

Reply:
I use Eclipse not IDEA.

Reply:
Looks like there's a checkstyle plugin for Eclipse too, but I cannot tell from their site if it provides import ordering according to checkstyle configuration.